xlators/cluster/ec/src/ec-inode-write.c

```diff
@@ -22,6 +22,8 @@ ec_update_writev_cbk(call_frame_t *frame, void *cookie, xlator_t *this,
                      struct iatt *postbuf, dict_t *xdata)
 {
     ec_fop_data_t *fop = cookie;
+    if (fop == NULL)
+        goto out;
```

Comment:
Earlier there was discussion for this bug and xavi suggested - 
`This is a bit more complex. If fop is NULL, it means that ec_writev() has failed to allocate a fop (op_ret will be -1). However this cbk is used for internal writes done as part of another fop, so we need to propagate this error to the parent. This cannot be done here for obvious reasons. I think we should modify ec_fop_data_allocate() so that it updates the parent when it's present and the fop cannot be created (unless flag EC_FOP_NO_PROPAGATE_ERROR is set).`